.evergreen/run-atlas-tests.sh

```diff
@@ -1,7 +1,8 @@
 #!/bin/bash
 
-# Don't trace to avoid secrets showing up in the logs
+# Exit on error and enable trace.
 set -o errexit
+set -o xtrace
```

Comment:
We can do this safely. We just need to remember not to print or use the secret variables in the shell script.